npm/src/clientfileprocessing/index.ts

```diff
@@ -44,11 +41,20 @@ export class ClientFileProcessing {
       progressBar.classList.remove("hide")
     }
 
-    if (progressBarElement && progressInformation.percentageProcessed < 50) {
-      progressBarElement.setAttribute(
-        "value",
-        progressInformation.percentageProcessed.toString()
-      )
+    if (progressBarElement) {
+      progressBarElement.setAttribute("value", weightedPercent.toString())
+    }
+  }
+
+  s3ProgressCallback(progressInformation: IProgressInformation) {
+    const weightedPercent = 50 + progressInformation.percentageProcessed / 2
+
+    const progressBarElement: HTMLDivElement | null = document.querySelector(
+      ".progress-display"
+    )
+
+    if (progressBarElement) {
+      progressBarElement.setAttribute("value", weightedPercent.toString())
     }
   }
 
```

Comment:
At line 64 the following parameter is passed into the processClientFile function: `callback: TProgressFunction`

This is no longer needed so should be removed.